py/picca/fitter2/chi2.py

```diff
@@ -0,0 +1,141 @@
+from __future__ import print_function
+import numpy as np
```

Comment:
To be consistent with the rest of picca, could you use scipy as sp?

Reply:
ok